show/main.py

```diff
@@ -3419,5 +3419,166 @@ def tunnel():
 
     click.echo(tabulate(table, header))
 
+#
+# 'system-health' command ("show system-health")
+#
+@cli.group(cls=AliasedGroup)
```

Comment:
Can you add an explicit `name='system-health'` argument here (there are other examples in this file)? As of CLick v7.0, they changed the default behavior of auto-translating group/command function names from using underscores to hyphens, and it affected us. If we explicitly provide the name here, we will not be surprised by any future changes.

Reply:
Thanks for the comment.
argument added.